### PR TITLE
Additional fixes

### PR DESCRIPTION
**Pull request by dmp42**
Couples of minor nits that where not addressed.

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -45,9 +45,9 @@ If the default configuration is not a sound basis for your usage, or if you are
...
 Typically, create a new configuration file from scratch, and call it `config.yml`, then:
 
-  docker run -d -p 5000:5000 --restart=always --name registry \
-    -v `pwd`/config.yml:/etc/docker/registry/config.yml \
-    registry:2
+    docker run -d -p 5000:5000 --restart=always --name registry \
+      -v `pwd`/config.yml:/etc/docker/registry/config.yml \
+      registry:2
 
 You can (and probably should) use [this a starting point](https://github.com/docker/distribution/blob/master/cmd/registry/config-example.yml).
```
Comment (RichardScothern) [inline docs/configuration.md]: this a -> this as a

**Comment by RichardScothern** [comment]
LGTM (with unrelated change fixed)

File: docs/deploying.md [hunk 2]
```diff
@@ -65,11 +65,11 @@ Move and/or rename your crt file to: `certs/domain.crt` - and your key file to:
...
-	docker run -d -p 5000:5000 --restart=always --name registry \
-	  -v `pwd`/certs:/certs \
-	  -e REGISTRY_HTTP_TLS_CERTIFICATE=/certs/domain.crt \
-	  -e REGISTRY_HTTP_TLS_KEY=/certs/domain.key \
-	  registry:2
+  	docker run -d -p 5000:5000 --restart=always --name registry \
+  	  -v `pwd`/certs:/certs \
+  	  -e REGISTRY_HTTP_TLS_CERTIFICATE=/certs/domain.crt \
+  	  -e REGISTRY_HTTP_TLS_KEY=/certs/domain.key \
+  	  registry:2
```
Comment (aaronlehmann) [inline docs/deploying.md]: Not sure this is correct - it's two spaces followed by a tab. I'd expect this to use four spaces. That said, github seems to render it properly, so maybe it's okay like this.

Comment (dmp42) [inline docs/deploying.md]: Ok. What do you think? tabs everywhere or spaces everywhere?

Comment (aaronlehmann) [inline docs/deploying.md]: I'm accustomed to four spaces at the beginning of the line to create these code blocks in markdown.

Comment (dmp42) [inline docs/deploying.md]: Spaces it is.

**Comment by aaronlehmann** [comment]
LGTM

**Pull request merged**